### ISSUE TITLE
Running on colab ..Unable to extract feature using c3d

### ISSUE THREAD

**Issue by suyash8991** [dolongbien/HumanBehaviorBKU#15]
# extract feature
%cd /content/C3D/C3D-v1.0/examples/c3d_feature_extraction
!sudo bash c3d_sport1m_feature_extraction_video.sh

This is the error I am getting  while executing the above code.
I0117 06:43:11.085855  4186 net.cpp:183] Collecting Learning Rate and Weight Decay.
I0117 06:43:11.085873  4186 net.cpp:176] Network initialization done.
I0117 06:43:11.085881  4186 net.cpp:177] Memory required for Data 5653289408
E0117 06:43:11.816212  4186 extract_image_features.cpp:81] Extracting features for 1 batches
F0117 06:43:13.534600  4186 vol2col.cu:75] Check failed: error == cudaSuccess (48 vs. 0)  no kernel image is available for execution on the device
*** Check failure stack trace: ***
c3d_sport1m_feature_extraction_video.sh: line 3:  4186 Aborted                 (core dumped) GLOG_logtosterr=1 ../../build/tools/extract_image_features.bin prototxt/c3d_sport1m_feature_extractor_video.prototxt conv3d_deepnetA_sport1m_iter_1900000 0 50 1 prototxt/output_list_video_prefix.txt fc7-1 fc6-1 prob

**Comment by aliakber366** [comment]
I ran into the same issue, @suyash8991 did you find any solution for this error?

**Comment by dolongbien** [comment]
Hi, make sure you were running under the GPU mode of Colab. Also, double-check the CUDA version and follow the right instruction on C3D notebook of this repo.

**Comment by mh-y** [comment]
Also, feature extractions for examples fail when already completed.
Please make sure that the feature has already been extracted.

**Comment by harshith7823** [comment]
I too got something similar -> **F0228 17:00:14.112013  6347 syncedmem.cpp:47] Check failed: error == cudaSuccess (46 vs. 0)  all CUDA-capable devices are busy or unavailable**
If anyone knows how to resolve this, pls let me know

**Comment by ericbyeric** [comment]
Try the following commands in the beginning of the code

# install cuda 10.2
!wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64/cuda-ubuntu1804.pin
!sudo mv cuda-ubuntu1804.pin /etc/apt/preferences.d/cuda-repository-pin-600
!sudo apt-key adv --fetch-keys https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64/7fa2af80.pub
!sudo add-apt-repository "deb https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64/ /"
!apt-get update
!apt-get -y install cuda-10.2

I resolved the same issue in this way
